scipy/signal/spectral.py

```diff
@@ -333,6 +335,8 @@ def welch(x, fs=1.0, window='hann', nperseg=None, noverlap=None, nfft=None,
         spectrum ('spectrum') where `Pxx` has units of V**2, if `x`
         is measured in V and `fs` is measured in Hz. Defaults to
         'density'
+    average : { 'mean', 'median' }, optional
+        Method to use when averaging periodograms. Defaults to 'mean'.
```

Comment:
In case someone has a long list of arguments, let's add this as the last argument instead of second to last (better backward compat)

Also, needs `.. versionadded:: 1.2.0` directive like:
```

    average : ...
        Method ...

        .. versionadded:: 1.2.0
"""
```